notebooks/00_quick_start/tfidf_covid.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<p>In general we follow a different structure when creating the code. If you look at SAR example, <a href="https://github.com/microsoft/recommenders/blob/master/notebooks/00_quick_start/sar_movielens.ipynb" target="_blank">https://github.com/microsoft/recommenders/blob/master/notebooks/00_quick_start/sar_movielens.ipynb</a>, there are some independent and structured steps: data prep, instantiate the algo with inputs of the algo, fit with the data using inputs for the training phase and then predict using inputs for the prediction phase. </p><p>The thinking of using this structure is, first it follows good python practices: <a href="https://github.com/Microsoft/Recommenders/wiki/Coding-Guidelines#the-zen-of-python" target="_blank">https://github.com/Microsoft/Recommenders/wiki/Coding-Guidelines#the-zen-of-python</a>, also, it is easier to reuse since everything is modular. It also is good for explaining the steps to an external audience.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1088/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
I have now created a `TfidfRecommender` class in `./reco_utils/recommender/tfidf/tfidf_utils.py` which more closely follows this structure. The quickstart notebook has been adapted to work with this new class.